README.md

```diff
@@ -1,18 +1,98 @@
-# Monorepo of Deeplearning4j
-
-Welcome to the new monorepo of Deeplearning4j that contains the source code for all the following projects, in addition to the original repository of Deeplearning4j moved to [deeplearning4j](deeplearning4j):
-
- * https://github.com/eclipse/deeplearning4j/tree/master/libnd4j
- * https://github.com/eclipse/deeplearning4j/tree/master/nd4j
- * https://github.com/eclipse/deeplearning4j/tree/master/datavec
- * https://github.com/eclipse/deeplearning4j/tree/master/arbiter
- * https://github.com/eclipse/deeplearning4j/tree/master/nd4s
- * https://github.com/eclipse/deeplearning4j/tree/master/rl4j
- * https://github.com/eclipse/deeplearning4j/tree/master/scalnet
- * https://github.com/eclipse/deeplearning4j/tree/master/pydl4j
- * https://github.com/eclipse/deeplearning4j/tree/master/jumpy
- * https://github.com/eclipse/deeplearning4j/tree/master/pydatavec
- 
+<p align="center">
+  <img src="https://www.zeljkoobrenovic.com/tools/tech/images/eclipse_deeplearning4j.png">
+</p>
+
+ [![Documentation](https://img.shields.io/badge/user-documentation-blue.svg)](https://deeplearning4j.konduit.ai/)
+[![Get help at the community forum](https://img.shields.io/badge/Get%20Help-Community%20Forum-blue)](https://community.konduit.ai/)
+[![javadoc](https://javadoc.io/badge2/org.deeplearning4j/deeplearning4j-common/DL4J%20API%20Doc.svg)](https://javadoc.io/doc/org.deeplearning4j/deeplearning4j-common)
```

Comment:
Wouldn't deeplearning4j-nn be the better module to link to?
deeplearning4j-common doesn't have any of the neural network stuff in it

Reply:
Yes, sorry, forgot to push the last update. I was wondering why only 2 where showing up.